src/psyclone/tests/domain/common/transformations/kernel_module_inline_trans_test.py

```diff
@@ -400,7 +400,7 @@ def test_module_inline_apply_transformation(tmpdir, fortran_writer):
 
     # And the import has been remove from both
     # check that the associated use no longer exists
-    assert 'use compute_cv_mod, only: compute_cv_code' not in code
+    assert 'use compute_cv_mod, only : compute_cv_code' not in code
     assert 'USE compute_cv_mod, ONLY: compute_cv_code' not in gen
```

Comment:
Not yours but this isn't a very good  test. It might be better just to check that `use compute_cv_mod` is not in `code.lower()`. Also, do we still need to test the `psy.gen` call?

Reply:
I agree with the assert.

I am unsure about the second, the "psy.gen" is what now does the copy/lower/psyir_backend, so its a valid way to use it. The second call to psy.gen just make sure the copy prevented changes to the original tree, otherwise we get symbols with indexing for the repeated kernels.